ui/tests/acceptance/dashboard-test.js

```diff
@@ -231,18 +231,21 @@ module('Acceptance | landing page dashboard', function (hooks) {
     });
 
     test('shows the correct actions and links associated with pki', async function (assert) {
-      await mountSecrets.enable('pki', 'pki');
+      const backend = 'pki-dashboard';
```

Comment:
Was getting errors about the issuer already existing, so I made this backend name a bit more unique